tests/my_land_analysis.py

```diff
@@ -0,0 +1,5 @@
+
+
+from freanalysis_land.land import LandAnalysisScript
+land = LandAnalysisScript()
+land.run_analysis("/work/a2p/lm4p2sc_catalog.json","/work/a2p/")
```

Comment:
take these lines and wrap them in a `def test_land_analysis_script():`. At the end of this function should be an `assert some_condition` that checks for success of `land.run_analysis`